src/utils/mev.ts

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios'
+import { ethers } from 'ethers'
+const logger = require('../services/logger')
+
+const bevUrl = 'https://bev-relay.kolibr.io'
```

Comment:
HTTP is good for POC and testing. 
But WS is more recommended, it will dramatically reduce latency.